packages/vats/test/test-vat-bank.js

```diff
@@ -158,7 +158,7 @@ test('communication', async t => {
   const balance = { address: 'agoricfoo', denom: 'ubld', amount: '92929' };
   const obj = { type: 'VBANK_BALANCE_UPDATE', updated: [balance] };
   t.assert(bankHandler);
-  await (bankHandler && E(bankHandler).fromBridge('bank', obj));
+  await E(bankHandler)?.fromBridge(obj);
```

Comment:
I don't see how `?.` could do anything meaningful for the return value of `E(x)`.

Reply:
It doesn't, but it shuts up TS. Will switch to a comment instead